website/pages/blog/styled-components-encapsulation.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introducing a new encapsulation method for Styled-Components with Babel
+title: Babel - Introducing a new encapsulation method for Styled-Components
```

Comment:
this once changes the meaning. 
Please use `New encapsulation method for Styled-Components with Babel`